plank/services/stock-ticker-service.go

```diff
@@ -173,10 +158,27 @@ func (ps *StockTickerService) OnServerShutdown() {
 	return
 }
 
-// GetRESTBridgeConfig returns nothing. this service is only available through
-// STOMP over WebSocket.
+// GetRESTBridgeConfig returns a config for a REST endpoint that performs the same action as the STOMP variant
+// except that there will be only one response instead of every 30 seconds.
 func (ps *StockTickerService) GetRESTBridgeConfig() []*service.RESTBridgeConfig {
-	return nil
+	return []*service.RESTBridgeConfig{
+		{
+			ServiceChannel: StockTickerServiceChannel,
+			Uri:            "/rest/stock-ticker/{symbol}",
+			Method:         http.MethodGet,
+			AllowHead:      true,
+			AllowOptions:   true,
+			FabricRequestBuilder: func(w http.ResponseWriter, r *http.Request) model.Request {
+				pathParams := mux.Vars(r)
+				return model.Request{
+					Id:                &uuid.UUID{},
+					Payload:           map[string]string{"symbol": pathParams["symbol"]},
```

Comment:
I think we will need to do some deep diving docs on this, passing path params, query / header params etc.

Reply:
yes, we'll need to thoroughly cover how to convert a http request into a Fabric request as this is a new concept to everyone